content/developer/cli.rst

```diff
@@ -292,9 +292,11 @@ Emails
 
 .. option:: --from-filter <address or domain>
 
-    Email address or domain that the mail server is configured for. If the 
-    email of the sender does not match this value, it might be encapsulated 
-    into ``mail.default.from``.
+    Define which email address the SMTP configuration will apply to. The field can be a domain name or
+    an entire email address, or it can remain empty. If the sender's email address does not match
+    this set filter, then the email will be encapsulated using a combination of the two system
+    parameters: ``mail.default.from`` and ``mail.catchall.domain``. For example, "Admin"
+    <admin\@example.com> => "Admin" <notifications\@mycompany.com>.
```

Comment:
Break slightly earlier starting at line 295 (otherwise char count @ 103, not 100).

```suggestion
    Define which email address the SMTP configuration will apply to. The field can be a domain name
    or an entire email address, or it can remain empty. If the sender's email address does not
    match this set filter, then the email will be encapsulated using a combination of the two
    system parameters: ``mail.default.from`` and ``mail.catchall.domain``. For example, "Admin"
    <admin\@example.com> => "Admin" <notifications\@mycompany.com>.
```